parser.y

```diff
@@ -552,11 +552,21 @@ import (
 	statsBuckets    "STATS_BUCKETS"
 	statsHealthy    "STATS_HEALTHY"
 	tidb		"TIDB"
-	tidbHJ		"TIDB_HJ"
```

Comment:
old hints should not be removed.

Reply:
Old hints will be treat as alias, the token will be same as `HASH_JOIN`.
You can check this out at the alias test.